Gems/Archive/Code/Include/Archive/ArchiveBus.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Archive/ArchiveTypeIds.h>
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace Archive
+{
+    class ArchiveRequests
```

Comment:
I'm guessing this EBus will be used at some point?

Reply:
Well this comes with the Gem Template and I don't have any need to use this Bus...